Themes/default/languages/Install.english.php

```diff
@@ -286,7 +286,7 @@
 $txt['upgrade_empty_errlog'] = 'Empty error log before upgrading';
 $txt['upgrade_delete_karma'] = 'Delete all karma settings and info from the DB';
 $txt['upgrade_stats_collection'] = 'Allow Simple Machines to Collect Basic Stats Monthly.';
-$txt['upgrade_stats_info'] = 'If enabled, this will allow Simple Machines to visit your site once a month to collect basic statistics. This will help us make decisions as to which configurations to optimise the software for. For more information please visit our <a href="https://www.simplemachines.org/about/stats.php" target="_blank" rel="noopener">info page';
+$txt['upgrade_stats_info'] = 'If enabled, this will allow Simple Machines to visit your site once a month to collect basic statistics. This will help us make decisions as to which configurations to optimise the software for. For more information please visit our %1$s.';
```

Comment:
I guess I wasn't quite clear earlier when I asked you to sprintf() the URL in. What I meant was to do this:
```
... For more information please visit our <a href="%1$s" target="_blank" rel="noopener">info page</a>.'
```
So, just the raw URL itself, not the whole HTML link.

Please change here and throughout.

Reply:
Oh, yes. I wanted to remove HTML from the text strings but that means translators won't be able to translate the words I passed through a var. I didn't think about it. Will change it later today.

Thank you :)